src/plugins/data/common/search/aggs/buckets/multi_terms_fn.ts

```diff
@@ -111,6 +111,12 @@ export const aggMultiTerms = (): FunctionDefinition => ({
         defaultMessage: 'Represents a custom label for this aggregation',
       }),
     },
+    separatorLabel: {
+      types: ['string'],
+      help: i18n.translate('data.search.aggs.buckets.multiTerms.separatorLabel.help', {
+        defaultMessage: 'The separator label used to join each term combination',
```

Comment:
curious: is this exposed to the user? Can we somehow change this separator?

Reply:
ATM the expression is not yet exposed to the final user, but it may be consumed by others, which may use another separator.
I thought that it won't be harmful to expose it as function argument and would provide some flexibility to others.